services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PinnedFlowV2Spec.groovy

```diff
@@ -173,7 +180,7 @@ class PinnedFlowV2Spec extends HealthCheckSpecification {
         errorDetails.errorMessage == "Could not update flow"
         errorDetails.errorDescription == "Flow flags are not valid, unable to process pinned protected flow"
 
-        and: "Cleanup: Delete the flow"
+        cleanup: "Cleanup: Delete the flow"
```

Comment:
`cleanup` is going to fail when `assumeTrue` is failed

let it be for now(it is not critical issue)